thunder/benchmarks/targets.py

```diff
@@ -155,11 +156,13 @@ def interpreter_fwd(module: Callable):
     torch_executor,
     torch_compile_executor,
     thunder_executor,
+    thunderfx_executor,
```

Comment:
What do we get by using this executor for all benchmarks in this file? All existing benchmarks work fine with plain Thunder.

Reply:
My idea is to add more information to the benchmarks, that then if not needed can be filtered out by using the `-k` filters. In particular this is useful for the torchbench benchmarks, but I thought that it is also useful for the other existing benchmarks don't you think so?

Reply:
Updated and splitted the executors to save on time on the execution of the benchmarks.